package.json

```diff
@@ -76,8 +76,8 @@
     "rimraf": "^3.0.2",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
-    "typedoc": "^0.23.15",
-    "typescript": "~4.8.4"
+    "typedoc": "^0.25.13",
+    "typescript": "~5.0.4"
```

Comment:
Just to be clear, are we upgrading to TypeScript just to verify that we are able to switch to `moduleResolution` = `nodenext` and therefore ensure that the upgrade to `@metamask/superstruct` doesn't produce the same errors that we were seeing in `core`?

Reply:
Replied here: https://github.com/MetaMask/snaps-registry/pull/613#discussion_r1633855974